cli/README.md

```diff
@@ -43,16 +42,16 @@ code-push access-key ls
 code-push access-key rm <accessKey>
 ```
 
-You can generate additional access keys for your account with a description tagged to each key. This lets you log in to CodePush and manage your apps without going through the 3rd party authentication step, e.g. from a CI environment.
+If you need additional keys, which can be used in a "headless" environment (particularly CI), without needing to give acces to your GitHub and/or Microsoft crendentials, you can run the following command to create one (along with a description of what it is for):
```

Comment:
Also this sentence feels kinda long.. maybe break it up "...crendentials. You can run..."